test/SparkDeployBase.t.sol

```diff
@@ -77,75 +77,86 @@ abstract contract SparkDeployBaseTest is Test {
     }
 
     function test_spark_deploy_poolAddressesProviderRegistry() public {
-        assertEq(poolAddressesProviderRegistry.owner(), admin);
         address[] memory providersList = poolAddressesProviderRegistry.getAddressesProvidersList();
-        assertEq(providersList.length, 1);
-        assertEq(providersList[0], address(poolAddressesProvider));
+
+        assertEq(poolAddressesProviderRegistry.owner(), admin);
+        assertEq(providersList.length,                  1);
+        assertEq(providersList[0],                      address(poolAddressesProvider));
+
         assertEq(poolAddressesProviderRegistry.getAddressesProviderAddressById(1),  address(poolAddressesProvider));
+
         assertEq(poolAddressesProviderRegistry.getAddressesProviderIdByAddress(address(poolAddressesProvider)), 1);
     }
 
     function test_spark_deploy_poolAddressesProvider() public {
-        assertEq(poolAddressesProvider.owner(), admin);
-        assertEq(poolAddressesProvider.getMarketId(), "Spark Protocol");
-        assertEq(poolAddressesProvider.getPool(), address(pool));
-        assertEq(poolAddressesProvider.getPoolConfigurator(), address(poolConfigurator));
-        assertEq(poolAddressesProvider.getPriceOracle(), address(aaveOracle));
-        assertEq(poolAddressesProvider.getACLManager(), address(aclManager));
-        assertEq(poolAddressesProvider.getACLAdmin(), admin);
+        assertEq(poolAddressesProvider.owner(),                  admin);
+        assertEq(poolAddressesProvider.getMarketId(),            "Spark Protocol");
+        assertEq(poolAddressesProvider.getPool(),                address(pool));
+        assertEq(poolAddressesProvider.getPoolConfigurator(),    address(poolConfigurator));
+        assertEq(poolAddressesProvider.getPriceOracle(),         address(aaveOracle));
+        assertEq(poolAddressesProvider.getACLManager(),          address(aclManager));
+        assertEq(poolAddressesProvider.getACLAdmin(),            admin);
         assertEq(poolAddressesProvider.getPriceOracleSentinel(), address(0));
-        assertEq(poolAddressesProvider.getPoolDataProvider(), address(protocolDataProvider));
+        assertEq(poolAddressesProvider.getPoolDataProvider(),    address(protocolDataProvider));
     }
 
     function test_spark_deploy_aclManager() public {
         // NOTE: Also verify that no other address than the admin address has any role (verify with events)
         assertEq(address(aclManager.ADDRESSES_PROVIDER()), address(poolAddressesProvider));
-        assertTrue(aclManager.hasRole(aclManager.DEFAULT_ADMIN_ROLE(), admin));
-        assertTrue(!aclManager.hasRole(aclManager.DEFAULT_ADMIN_ROLE(), deployer));
-        assertEq(aclManager.getRoleAdmin(aclManager.POOL_ADMIN_ROLE()), aclManager.DEFAULT_ADMIN_ROLE());
-        assertTrue(aclManager.hasRole(aclManager.POOL_ADMIN_ROLE(), admin));
+
+        bytes32 defaultAdmin = aclManager.DEFAULT_ADMIN_ROLE();
```

Comment:
Similarly probably makes sense to cache the `aclManager.POOL_ADMIN_ROLE()` as well.